scoper.inc.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+require_once sprintf('%1$s/tests/stub/external/IdentifierExtractor.php', __DIR__);
+require_once sprintf('%1$s/tests/stub/external/RemovePrefixPatcher.php', __DIR__);
+
+use pxlrbt\PhpScoper\PrefixRemover\IdentifierExtractor;
+use pxlrbt\PhpScoper\PrefixRemover\RemovePrefixPatcher;
+use Isolated\Symfony\Component\Finder\Finder;
+
+// Will likely need updating if a different version of PHPScoper is used
+class_alias('_HumbugBoxb6056d4e1e8c\PhpParser\ParserFactory', 'PhpParser\ParserFactory');
+
+return [
+    // The prefix configuration. If a non null value will be used, a random prefix will be generated.
+    'prefix' => 'MollieWc',
+
+    // By default when running php-scoper add-prefix, it will prefix all relevant code found in the current working
+    // directory. You can however define which files should be scoped by defining a collection of Finders in the
+    // following configuration key.
+    //
+    // For more see: https://github.com/humbug/php-scoper#finders-and-paths
+    'finders' => [
+        Finder::create()->files()->in('src'),
+        Finder::create()
+            ->files()
+            ->ignoreVCS(true)
+            ->notName('/LICENSE|.*\\.md|.*\\.dist|Makefile|composer\\.json|composer\\.lock/')
+            ->exclude([
+                'doc',
+                'test',
+                'test_old',
+                'tests',
+                'Tests',
+                'vendor-bin',
+            ])
+            ->in('vendor'),
+    ],
+
+    // Whitelists a list of files. Unlike the other whitelist related features, this one is about completely leaving
+    // a file untouched.
+    // Paths are relative to the configuration file unless if they are already absolute
+    'files-whitelist' => [
+        'src/a-whitelisted-file.php',
```

Comment:
I don't understand where is this file from. It looks like an example from docs. Should it be like so?